File: Week1~Python/Tas-Kagit-Makas.py
# Taş-kağıt-makas
# Kodunuzu bu dosya içine yazınız
"""
Kodunuzun düzgün çalışması için
input metodunun içindeki metni boş bırakın
input('')
şeklinde...
"""
a = input('')
b = input('')

if(a == b):
    print("berabere")
elif(a == "taş" and b == "kağıt" or a == "kağıt" and b == "makas" or a == "makas" and b == "taş"):
    print("ikinci oyuncu kazandı")
else:
    print("birinci oyuncu kazandı")

File: Week4~Ruby/Tas-Kagit-Makas.rb
# Taş-kağıt-makas
# Kodunuzu bu dosya içine yazınız
=begin
Kodunuzun düzgün çalışması için
input metodunun içindeki metni boş bırakın
gets.chomp
yeterli...
=end
i1 = gets.chomp
i2 = gets.chomp

if i1 == i2
    puts "berabere"
elsif (i1 == "taş" && i2 == "makas") || (i1 == "makas" && i2 == "kağıt") || (i1 == "kağıt" && i2 == "taş")
    puts "birinci oyuncu kazandı"
else
    puts "ikinci oyuncu kazandı"
end